### PR TITLE
Added logging in json

### PR DESCRIPTION
**Pull request by filiphaftek**
Logging in json format, to stdout, with all required fields explicitly added - ready for logstash etc.
Sample output:
```
rabbitamazonforwarder_1  | time="2017-09-29T07:07:31Z" level=info msg="Starting http server"
rabbitamazonforwarder_1  | time="2017-09-29T07:07:31Z" level=info msg="Starting connecting consumer" exchangeName=test-topic queueName=test-queue
rabbitamazonforwarder_1  | time="2017-09-29T07:07:32Z" level=info msg="Started forwarding messages" consumerName=test-rabbit forwarderName=test-lambda
rabbitamazonforwarder_1  | time="2017-09-29T07:08:00Z" level=info msg="Message to forward" consumerName=test-rabbit messageID=
rabbitamazonforwarder_1  | time="2017-09-29T07:08:01Z" level=error msg="Could not forward message" error="ResourceNotFoundException: Function not found: arn:aws:lambda:eu-west-1:555911484621:function:test-forwarder-lambda\n\tstatus code: 404, request id: ee6ed94c-a4e4-11e7-b43a-1b49e553dcd0" forwarderName=test-lambda
```

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-
 	"github.com/AirHelp/rabbit-amazon-forwarder/mapping"
 	"github.com/AirHelp/rabbit-amazon-forwarder/supervisor"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"os"
 )
 
 func main() {
+	log.SetFormatter(&log.TextFormatter{})
```
Comment (kszarek) [inline server.go]: since you want a json format, maybe you'd use `JSONFormatter` ?

File: lambda/forwarder.go [hunk 2]
```diff
@@ -33,7 +32,7 @@ func CreateForwarder(entry config.AmazonEntry, lambdaClient ...lambdaiface.Lambd
 		client = lambda.New(session.Must(session.NewSession()))
 	}
 	forwarder := Forwarder{entry.Name, client, entry.Target}
-	log.Print("Created forwarder: ", forwarder.Name())
+	log.WithFields(log.Fields{"forwarderName": forwarder.Name()}).Info("Created forwarder")
```
Comment (swojtasiak) [inline lambda/forwarder.go]: FYI. In cases where only one field is defined, you can use the shorter form: WithField("forwarderName", forwarder.Name).

File: server.go (hunk 1 above)
Comment (filiphaftek) [reply server.go]: Ma bad, I've changed it just before commit to test colouring with simple text formatter.

File: lambda/forwarder.go (hunk 2 above)
Comment (filiphaftek) [reply lambda/forwarder.go]: good hint, thx.

File: server.go [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
+	log.SetOutput(os.Stdout)
```
Comment (swojtasiak) [inline server.go]: We should probably add a way to configure log level here.

**Pull request merged**